rosidl_typesupport_introspection_c/resource/srv__type_support.c.em

```diff
@@ -46,22 +57,42 @@ static rosidl_typesupport_introspection_c__ServiceMembers @(function_prefix)__@(
   // these two fields are initialized below on the first access
   NULL,  // request message
   // @(function_prefix)__@(service.request_message.structure.namespaced_type.name)_message_type_support_handle,
-  NULL  // response message
+  NULL,  // response message
+  // @(function_prefix)__@(service.response_message.structure.namespaced_type.name)_message_type_support_handle
+  NULL  // event_message
   // @(function_prefix)__@(service.response_message.structure.namespaced_type.name)_message_type_support_handle
 };
 
+@{event_type = '__'.join([package_name, *interface_path.parents[0].parts, service.namespaced_type.name]) + SERVICE_EVENT_MESSAGE_SUFFIX}@
+@{request_type = '__'.join([package_name, *interface_path.parents[0].parts, service.namespaced_type.name]) + SERVICE_REQUEST_MESSAGE_SUFFIX}@
+@{response_type = '__'.join([package_name, *interface_path.parents[0].parts, service.namespaced_type.name]) + SERVICE_RESPONSE_MESSAGE_SUFFIX}@
+
 static rosidl_service_type_support_t @(function_prefix)__@(service.namespaced_type.name)_service_type_support_handle = {
   0,
   &@(function_prefix)__@(service.namespaced_type.name)_service_members,
   get_service_typesupport_handle_function,
+  ROSIDL_TYPESUPPORT_INTERFACE__SERVICE_CREATE_EVENT_MESSAGE_SYMBOL_NAME(
+    rosidl_typesupport_c,
+    @(',\n    '.join(service.namespaced_type.namespaced_name()))
+  ),
+  ROSIDL_TYPESUPPORT_INTERFACE__SERVICE_DESTROY_EVENT_MESSAGE_SYMBOL_NAME(
+    rosidl_typesupport_c,
+    @(',\n    '.join(service.namespaced_type.namespaced_name()))
+  ),
+  &@(message_function_prefix)__@(service.namespaced_type.name)@(SERVICE_EVENT_MESSAGE_SUFFIX)_message_type_support_handle,
 };
+// rcl_interfaces__srv__GetParameters_Event__rosidl_typesupport_introspection_c__GetParameters_Event_message_type_support_handle
+// rcl_interfaces__srv__detail__get_parameters__rosidl_typesupport_introspection_c__GetParameters_Event_message_type_support_handle,
```

Comment:
What are these comments for?

Reply:
It was an old alternative that I commented out and I forgot to delete.
Fixed in https://github.com/ros2/rosidl/pull/700/commits/621facc30f273fc393f215706166b00b1ad812ed.